containers/vm-provisioning/ansible/roles/osh-worker-role/tasks/main.yml

```diff
@@ -76,10 +76,10 @@
 
 # Workaround for https://lists.fedoraproject.org/archives/list/openscanhub@lists.fedoraproject.org/thread/GY2XQBNNDHR5VHGA24MBAUBKNAJPUDZW/
 # TODO: Remove when latest gcc is released
-- name: Copy /etc/mock/site-defaults.cfg
+- name: Copy /etc/mock/fedora-rawhide-gcc-latest-x86_64.cfg
```

Comment:
It is not possible to override existing `dnf.conf` by default as it is [overwritten](https://github.com/rpm-software-management/mock/blob/4e35c40fa6d25d86b5c6097f24dcfdefab0bd97a/mock-core-configs/etc/mock/templates/fedora-rawhide.tpl#L20) by corresponding mock configurations, so I have create a new configuration and profile for `gcc-latest`.